spec/requests/workflows/single_workflow_for_object_spec.rb

```diff
@@ -3,77 +3,38 @@
 require 'rails_helper'
 
 RSpec.describe 'Get a single workflow for an object', type: :request do
-  context 'when repo is supplied (deprecated)' do
```

Comment:
AFAICT, there was no different in the specs inside the two contexts (with vs. without the repo param)